src/set_theory/game/pgame.lean

```diff
@@ -373,6 +385,12 @@ move_left_lf_of_le i le_rfl
 theorem lf_move_right {x : pgame} (j) : x ⧏ x.move_right j :=
 lf_move_right_of_le j le_rfl
 
+theorem lf_mk {xl xr} (xL : xl → pgame) (xR i) : xL i ⧏ mk xl xr xL xR :=
+@move_left_lf (mk _ _ _ _) i
+
+theorem mk_lf {xl xr} (xL xR) : ∀ j, mk xl xr xL xR ⧏ xR j :=
+@lf_move_right (mk _ _ _ _)
```

Comment:
Could these two statements be made more symmetric? LGTM otherwise, thanks!

Reply:
Sure thing, and thanks a lot for the review!